cfgov/jinja2/v1/_includes/organisms/filterable-list.html

```diff
@@ -0,0 +1,200 @@
+{# ==========================================================================
+
+   Filterable list
+
+   ==========================================================================
+
+   Description:
+
+   Create a filter list that includes controls and results when given:
+
+   value: Data object from an instance of the FilterableList StreamField block.
+
+   value.no_posts_message: Message to show in "no posts" state.
+
+   value.no_posts_explanation: More info to show in "no posts" state.
+
+   value.form_type: What type of form this is.
+
+   value.categories.page_type: Page type of the filterable list control.
```

Comment:
```suggestion
   value.categories.page_type: What kind of pages are being filtered, e.g. 'final-rule', 'blog'. 
                               This is used to determine what categories to allow filtering on,
                               as well as how the post preview is displayed.
```

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/4771/commits/dee1b3129ec90e3585884c6f650c6d448aaf72be
I wrapped it a little differently to fit in the 80 character the rest of the comments are in.